framework/src/modules/chain/blocks/fork_choice_rule.js

```diff
@@ -19,35 +19,30 @@ const forgingSlot = (slots, block) => slots.getSlotNumber(block.timestamp);
  * This is evaluated in the first line.
  * @param slots
  * @param lastAppliedBlock
- * @param lastReceivedAndAppliedBlock
  * @return {boolean}
  */
 const islastAppliedBlockReceivedWithinForgingSlot = (
 	slots,
-	lastAppliedBlock,
-	lastReceivedAndAppliedBlock
+	lastAppliedBlock
 ) => {
-	if (lastAppliedBlock.id !== lastReceivedAndAppliedBlock.id) return true;
+	// If the blockk doesn't have the property `receivedAt` it meants it was forged
```

Comment:
typo: `blockk`, `meants`  :D https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker :D 🖖

Reply:
I don't use VSCode hehe but thanks for the good catch. Solved :)